cirq/contrib/quirk/cells/frequency_space_cells.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2018 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Iterator, Callable
+
+import sympy
+
+import cirq
+from cirq.contrib.quirk.cells.cell import (
+    CellMaker,
+    ExplicitOperationsCell,
+    CELL_SIZES,
+)
+
+
+def generate_all_frequency_space_cell_makers() -> Iterator[CellMaker]:
+    # Frequency space.
+    yield from reg_family("QFT", lambda n: cirq.QuantumFourierTransformGate(n))
+    yield from reg_family(
+        "QFT†", lambda n: cirq.inverse(cirq.QuantumFourierTransformGate(n)))
+    yield from reg_family(
+        "PhaseGradient", lambda n: cirq.PhaseGradientGate(num_qubits=n,
+                                                          exponent=0.5))
+    yield from reg_family(
+        "PhaseUngradient", lambda n: cirq.PhaseGradientGate(num_qubits=n,
```

Comment:
nit: This name is weird. "PhaseGradient†" would be both clearer and consistent with the QFT cell above.

Reply:
Too late to change it now, this is the id used in Quirk.
(I was not consistent over time with the ids, in many many ways.)

Reply:
This is forced on the Quirk side. I was not consistent with gate ids over time, in many ways.